public/js/utils/source-map-worker.js

```diff
@@ -22,21 +33,21 @@ function enableSourceMaps() {
   sourceMapsEnabled = true;
 }
 
-function _resolveSourceMapURL(source) {
-  if (path.isURL(source.sourceMapURL) || !source.url) {
+function _resolveSourceMapURL(source: Source) {
+  const { url, sourceMapURL } = source;
```

Comment:
Just a reminder: (since other comment was on an old commit, I believe) don't destructure here and follow what the code did before.